docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+services:
+  rabbitmq:
+    image: rabbitmq:3.7-management
+    environment:
+      RABBITMQ_DEFAULT_USER: iib
+      RABBITMQ_DEFAULT_PASS: iib
+    ports:
+      # The RabbitMQ management console
+      - 8081:15672
+
+  iib-worker:
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile-workers
+    # Override the default command so that Celery auto-reloads on code changes
+    command:
+    - /bin/bash
+    - -c
+    - >-
+      pip3 install watchdog[watchmedo] &&
```

Comment:
shouldn't it just be `pip3 install watchdog`? Am I missing something?

Reply:
Ah, got it. nvm :)

Reply:
By doing `watchdog[watchmedo]`, it installs `watchdog` with the extra requirements required to use the `watchmedo` feature.

You can see that this is defined here:
https://github.com/gorakhargosh/watchdog/blob/b752e371792eadf4d1e79cdcc15c40f3539bcb96/setup.cfg#L7-L10

Reply:
Ah, I learned something today. thanks!